web/src/layouts/MainLayout/MainLayout.js

```diff
@@ -99,19 +98,34 @@ const MainLayout = ({ children }) => {
                 <Svg name="plus" className="text-gray-700 w-full h-full" />
               )}
             </li>
-            <li className="h-10 w-10 border-1 rounded-full border-indigo-300 text-indigo-200">
-              <div aria-describedby={popoverId} onMouseOver={togglePopover}>
-                {!loading && (
-                  <ImageUploader
-                    className="rounded-full object-cover"
-                    onImageUpload={() => {}}
-                    aspectRatio={1}
-                    imageUrl={data?.user?.image}
-                    width={80}
-                  />
-                )}
-              </div>
-            </li>
+            {isAuthenticated ? (
+              <li
+                className="h-10 w-10 border-2 rounded-full border-indigo-300 text-indigo-200"
+                aria-describedby={popoverId}
+              >
+                <button className="w-full h-full" onClick={togglePopover}>
+                  {!loading && (
+                    <ImageUploader
+                      className="rounded-full object-cover"
+                      onImageUpload={() => {}}
+                      aspectRatio={1}
+                      imageUrl={data?.user?.image}
+                      width={80}
+                    />
+                  )}
+                </button>
+              </li>
+            ) : (
+              <li>
+                <a
+                  href="#"
+                  className="text-indigo-200 font-semibold underline"
+                  onClick={logIn}
+                >
+                  Sign in/up
+                </a>
+              </li>
```

Comment:
Added this back as there is not gravatar to show when the user is not signed in.
Also makes it clearer for users who are yet to signup
![image](https://user-images.githubusercontent.com/29681384/99449957-30680280-2974-11eb-8d6c-46ddeeaf4098.png)

Reply:
When the user is not loggedin we can also make some think like this. 

![Screenshot (21)](https://user-images.githubusercontent.com/41328269/99491118-328c8a00-2991-11eb-827e-305a89be043b.png)

and when the user would be loggedin then we can show his own avatar?
What say @Irev-Dev

Reply:
I think I'd rather it be an explicit call to action, to sign up, best to have it as clear as possible.